src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -826,7 +829,7 @@ public async Task PostAsync_ManyDifferentRequestHeaders_SentCorrectly()
                     request.Headers.TE.Add(new TransferCodingWithQualityHeaderValue("trailers"));
                     request.Headers.TE.Add(new TransferCodingWithQualityHeaderValue("deflate"));
                     request.Headers.Trailer.Add("MyTrailer");
-                    request.Headers.TransferEncoding.Add(new TransferCodingHeaderValue("chunked"));
+                    if (!IsCurlHandler) request.Headers.TransferEncoding.Add(new TransferCodingHeaderValue("chunked"));
```

Comment:
Why this change?

Reply:
I could not get curl stable with HTTP2 and chunked encoding. I could not reproduce it now, so I move it back as it was before.